lib/libutee/tee_api_property.c

```diff
@@ -201,7 +201,7 @@ TEE_Result TEE_GetPropertyAsString(TEE_PropSetHandle propsetOrEnumerator,
 {
 	TEE_Result res;
 	size_t l;
-	enum user_ta_prop_type type;
+	enum user_ta_prop_type type = 0;
```

Comment:
IIUC the important thing here is that the initial value be `!= USER_TA_PROP_TYPE_BINARY_BLOCK`. Should we introduce `USER_TA_PROP_TYPE_INVALID`? Or maybe just add a `static_assert(USER_TA_PROP_TYPE_BINARY_BLOCK != 0);`?

Reply:
I went for `USER_TA_PROP_TYPE_INVALID`